components/layout_2020/style_ext.rs

```diff
@@ -335,6 +347,87 @@ impl ComputedValuesExt for ComputedValues {
         )
     }
 
+    fn outer_inline(
```

Comment:
This move feels conceptually wrong to me. On the method receiver type: styles don’t have content sizes, boxes do. On the module location (which is independent): this code belongs in the `sizing` module.

Reply:
That method living on `BoxContentSizes` look even more wrong to me given it seldomly does anything with `self` there. "There are styles, and those styles can clamp an inline size, or a content size if there are no inline size" makes *way* more sense to me and better compartmentalize the code.

Reply:
And given you can't split a trait in multiple parts, this needs to be in `style_ext`.

Reply:
To further explain what I mean: styles don't have content sizes, but they definitely are what owns the size extremums, and the box-sizing property, and often a non-auto inline size. When there is a non-auto inline size, the `BoxContentSizes` value isn't even looked at. That means to me that it shouldn't be a method on `BoxContentSizes`.

Reply:
How about making it a free function in the `sizing` module? `style_ext` already feels like a kitchen sink to me.

Reply:
That is totally fair, will do that, absolutely no damn clue how I didn't think of that on my own.